pagopa/io-app
# feat: [IOBP-124] Add new layout in pagoPA barcode scan screen

## Short description
This PR adds the new barcode scan layout in the pagoPA notice payment screen.


https://github.com/pagopa/io-app/assets/6160324/babf280e-775c-4036-8e21-b565c09c9f6e


## List of changes proposed in this pull request
- ts/features/walletV3/payment/screens/WalletPaymentBarcodeScanScreen.tsx: new pagoPA barcode scan screen
- ts/features/walletV3/payment/navigation/navigator.tsx: added payment navigator
- ts/screens/wallet/payment/ScanQrCodeScreen.tsx: deprecated `ScanQrCodeScreen` component
- ts/screens/wallet/WalletHomeScreen.tsx: added navigation to the new pagoPA barcode scan screen

## How to test
Enable the Experimental Design System from the profile screen. Using the `io-dev-server`, from the wallet section, click on "Pay notice" button. Check that the screen is displayed correctly and there is the ability to scan, upload and digit a payment notice code.
